ruby/example_code/elasticbeanstalk/list_apps.rb

```diff
@@ -0,0 +1,50 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+require "aws-sdk-elasticbeanstalk"
+require "logger"
+
+# snippet-start:[eb.Ruby.listApps]
+# Class to manage Elastic Beanstalk applications
+class ElasticBeanstalkManager
+  def initialize(eb_client)
+    @eb_client = eb_client
+    @logger = Logger.new($stdout)
```

Comment:
May want to accept the logger as a param on initialize with a default. eg:
```ruby
def initialize(eb_client, logger: Logger.new($stdout))

# or
def initialize(eb_client, logger)
   @logger = logger | Logger.new($stdout)
```

Reply:
Added this to a dozen newly-added examples.